package.json

```diff
@@ -42,7 +42,7 @@
     "postinstall-postinstall": "^2.1.0",
     "react": "17.0.2",
     "react-i18next": "^11.5.0",
-    "react-native": "^0.65.1",
+    "react-native": "0.66.0",
```

Comment:
I think you can save some time by upgrading to the latest patch version of each step, which would require just 1 upgrade per minor version, instead of dozens of them.

Reply:
It does make sense.
Surprisingly all the changes made to [ios/mapswipe.xcodeproj/project.pbxproj](https://github.com/mapswipe/mapswipe/pull/746/files#diff-8fd159f5cf53c9ced0c8a959f2a9dac2490e172c9f3fc3966c73f6edd5b8f061) in 0.66.0 was removed in 0.66.5
So, it should be a bit easier as well.

Reply:
Yes, that `pbxproj` file is a disaster to update. Well done fixing it!